website/components/_temp-enable-hidden-pages/index.js

```diff
@@ -0,0 +1,85 @@
+// Imports below are used in server-side only
+import fs from 'fs'
+import path from 'path'
+import {
+  generateStaticPaths as docsPageStaticPaths,
+  generateStaticProps as docsPageStaticProps,
+} from '@hashicorp/react-docs-page/server'
+
+/**
+ * DEBT
+ * This is a short term hotfix for "hidden" docs-sidenav items.
```

Comment:
Hopefully this comment covers the context here - once we come up with an alternate solution (eg a modification to `docs-page/server.js` or to `docs-sidenav` or something), I think this will be relatively straightforward to switch back out.

With that in mind, I think this same solution can be applied to other sites where unlinked content is an issue.